SConstruct

```diff
@@ -64,53 +64,57 @@ AddOption('--no-test',
           default=os.path.islink(Dir('#laika/').abspath),
           help='skip building test files')
 
-## Architecture name breakdown (arch)
-## - larch64: linux tici aarch64
-## - aarch64: linux pc aarch64
-## - x86_64:  linux pc x64
-## - Darwin:  mac x64 or arm64
-real_arch = arch = subprocess.check_output(["uname", "-m"], encoding='utf8').rstrip()
-if platform.system() == "Darwin":
-  arch = "Darwin"
+# *** Target and Architecture
+
+## Target name breakdown (target)
+## - agnos-aarch64: linux tici aarch64
```

Comment:
@adeebshihadeh what do you think about replacing slightly ambiguous `larch64` with `agnos-aarch64` for TICI target?